server/src-lib/Hasura/GraphQL/Resolve/Mutation.hs

```diff
@@ -51,6 +51,7 @@ convertMutResp ty selSet =
       UVPG annPGVal -> UVSQL <$> txtConverter annPGVal
       UVSessVar colTy sessVar -> pure $ UVSessVar colTy sessVar
       UVSQL sqlExp -> pure $ UVSQL sqlExp
+      UVSession -> pure UVSession
```

Comment:
Here we're intentionally encoding input values in `returning` field to Postgres literals but not prepared values (as like `$<Int>`) since mutations can be performed in two separate queries (See https://github.com/hasura/graphql-engine/commit/5f274b552717d107553368fec440eb7137e830fb) if `returning` has nested fields selected. It is hard to maintain the sequence for prepared values for `returning` field. Though the function is not informative, I'll make it so.